components/fxa-client/src/profile.rs

```diff
@@ -9,6 +9,12 @@ use crate::{errors::*, scopes, util, CachedResponse, FirefoxAccount};
 const PROFILE_FRESHNESS_THRESHOLD: u64 = 120000; // 2 minutes
 
 impl FirefoxAccount {
+    /// Fetch the profile for the user.
+    /// This method will error-out if the `profile` scope is not
+    /// authorized for the current refresh token or if we don't hold any.
```

Comment:
nit: "or if we don't hold any" is not very clear here as to any *what*; I suggest something like "or if we do not have a valid refresh token" for clarity.